js/server.js

```diff
@@ -38,7 +38,7 @@ var Server = function(config, callback) {
 			res.status(403).send("This device is not allowed to access your mirror. <br> Please check your config.js or config.js.sample to change this.");
 		});
 	});
-	app.use(helmet());
+	app.use(helmet(config.helmetOptions));
```

Comment:
I would not go for straight `config.helmetOptions`, but instead specify a helmet config with some sane defaults if the `helmetOptions` key is empty.

Something like this:

```javascript
if(config.helmetOptions) {
    app.use(helmet(config.helmetOptions));
} else {
    app.use(helmet({
        dnsPrefetchControl: true,
        frameguard: false,
        hidePoweredBy: true,
        hsts: false,
        ieNoOpen: false,
        noSniff: true,
        xssFilter: true
    });
}